$Id$

This patch is required to make procmail cross compile - it tries to run some
automatic tests which obviously do not work in a cross compiled environment.

--- procmail-3.22.orig/autoconf.h	1970-01-01 00:00:00.000000000 +0100
+++ procmail-3.22/autoconf.h	2006-07-27 07:50:48.000000000 +0200
@@ -0,0 +1,21 @@
+/* This file was automagically generated by autoconf */
+
+/* 5 moves in 64 steps of size 16384 when reallocing */
+#define NOpw_class
+#define NOstrlcat
+#define NOsetrgid
+#define MAX_argc 7588
+/* Your system's strstr() is 1.12 times FASTER than my C-routine */
+#define UDP_protocolno 17
+#define BIFF_serviceport "512"
+#define IP_localhost {127,0,0,1}
+#define MAILSPOOLDIR "/var/spool/mail/"
+#define SENDMAIL "/usr/sbin/sendmail"
+#define CF_no_procmail_yet
+#define buggy_SENDMAIL
+#define defPATH "PATH=$HOME/bin:/bin:/usr/bin:/usr/local/bin:/usr/bin/X11"
+#define defSPATH "PATH=/bin:/usr/bin:/usr/local/bin:/usr/bin/X11"
+#define PM_VERSION "3.22"
+/* Hotwire LOCKINGTEST=100 */
+/* Procmail will lock via: dotlocking, fcntl() */
+/* autoconf completed */
